opentok/opentok-meet
# OPENTOK-48348: Migrate to new redis service

#### What is this PR doing?
Our current redis provider, [redistogo](https://redistogo.com/), is shutting down, so we need to migrate to another provider: heroku's.

#### How should this be manually tested?
To confirm that you can create sessions and do pub/sub, first make changes below.
```
bash-3.2$ git diff
diff --git a/app.js b/app.js
index 095b424..523fcb0 100644
--- a/app.js
+++ b/app.js
@@ -43,6 +43,9 @@ if (config.clientId) {
 
 let redisClient;
 
+// We hardcode this so mimic this running from our heroku instance
+process.env.HEROKU_REDIS_URL = 'Pssst, look in heroku config for this value';
+
 if (process.env.HEROKU_REDIS_URL) {
   redisClient = redis.createClient({
     url: process.env.REDIS_URL,
bash-3.2$ 
```

After that:
* Make sure you have a valid `config.json` file in the working directory (you can use `config.json.sample` as a template)
* Build it: `npm install && npm run build`
* Open up http://localhost:3000 and do pub/sub

You did it! 💪 
Now, grab a 🍺 or two or three and celebrate! 🎉 

#### What are the relevant tickets?
https://jira.vonage.com/browse/OPENTOK-48348
